server/metrics/metrics.go

```diff
@@ -186,6 +189,14 @@ const (
 
 	/// The TreeCache status: hit/miss/invalid_entry.
 	TreeCacheLookupStatus = "status"
+
+	/// For firecracker remote execution runners, describes the snapshot
+	/// sharing status (Ex. 'disabled' or 'local_sharing_enabled')
+	SnapshotSharingStatus = "snapshot_sharing_status"
+
+	/// For remote execution runners, describes the recycling status (Ex.
+	/// 'clean' if the runner is not recycled or 'recycled')
+	RecycledRunnerStatus = "recycled_runner_status"
```

Comment:
ugh sorry I should have waited for your PR to be merged before merging https://github.com/buildbuddy-io/buildbuddy/pull/4668

```suggestion
	// For firecracker remote execution runners, describes the snapshot
	// sharing status (Ex. 'disabled' or 'local_sharing_enabled')
	SnapshotSharingStatus = "snapshot_sharing_status"

	// For remote execution runners, describes the recycling status (Ex.
	// 'clean' if the runner is not recycled or 'recycled')
	RecycledRunnerStatus = "recycled_runner_status"
```